vgo-core/build.gradle.kts

```diff
@@ -1,6 +1,7 @@
 plugins {
     id("org.jetbrains.kotlin.jvm")
     id("com.vanniktech.maven.publish")
+    id("org.jetbrains.kotlinx.binary-compatibility-validator") version "0.16.3"
```

Comment:
The validation check is hooked into the build task automatically. No workflow change is necessary.

```
% ./gradlew build -m | rg apiCheck
:vgo-core:apiCheck SKIPPED
```